internal/pkg/deploy/cloudformation/stack/env_integration_test.go

```diff
@@ -70,7 +70,84 @@ observability:
 			}(),
 			wantedFileName: "template-with-imported-certs-observability.yml",
 		},
-
+		"generate template with default access logs": {
```

Comment:
Yeah I think this point is valid. What I will do is instead of putting everything in one test case I will just create one test case separately for access logs and put all the possible configs related to the access logs in it instead of creating separate test for each use case. And I will make sure I will take care of the other edge cases in my unit test locally and through `validate_env.test`

Reply:
I was going to remove them at the end because I was using them for the testing purpose until I address all other feedback comments. 
For now I have removed 2 tests related to access logs and kept only one which tests `advancedAccessLogs` and added default access logs as a part of one of the existing tests.

For now I will keep it like this and later we can create only one test including all the fields of the env manifest as Wanxian mentioned. But right now doing so is a bit heavy work for me as a part of this feature.